src/modules/esl-traversing-query/core/esl-traversing-query.ts

```diff
@@ -95,8 +95,35 @@ export class TraversingQuery {
     }
     return uniq(result);
   }
+  /** Split multiple queries separated by comma (respects query brackets) */
+  /** This can be solved by RegEx /(?<!\([^\)]*),(?![^\(]*\))/g)/, when the WebKit browser implements this feature  */
```

Comment:
Maybe replace it with
```suggestion
  // This can be solved by RegEx /(?<!\([^\)]*),(?![^\(]*\))/g)/, when the WebKit browser implements this feature
```
to avoid `tsdoc-malformed-html-name` warning?